<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cangshan</class>
 <widget class="QDialog" name="Cangshan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1071</width>
    <height>728</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1071</width>
    <height>728</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1071</width>
    <height>728</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>无涯</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Menu.qrc">
    <normaloff>:/icon/Resources/无涯.png</normaloff>:/icon/Resources/无涯.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/image/Resources/cangshan.png);</string>
  </property>
  <widget class="QPushButton" name="btnDadian">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>340</y>
     <width>181</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/other/Resources/透明背景.png);
border-image: url(:/other/Resources/边框2.png);
color: rgb(255, 255, 255);
font: 32pt &quot;隶书&quot;;</string>
   </property>
   <property name="text">
    <string>大殿</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMishi">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>130</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/other/Resources/透明背景.png);
border-image: url(:/other/Resources/边框2.png);
color: rgb(255, 255, 255);
font: 26pt &quot;隶书&quot;;</string>
   </property>
   <property name="text">
    <string>密室</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblText">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>510</y>
     <width>971</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/other/Resources/透明背景.png);
color: rgb(255, 255, 255);
font: 16pt &quot;隶书&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOut">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>630</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/other/Resources/透明背景.png);
border-image: url(:/other/Resources/边框2.png);
color: rgb(255, 255, 255);
font: 26pt &quot;隶书&quot;;</string>
   </property>
   <property name="text">
    <string>离开</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Menu.qrc"/>
 </resources>
 <connections/>
</ui>
